sadx-input-mod/SADXKeyboard.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include <SADXStructs.h>
+
+struct KeyboardKey
+{
+	char held;
+	char old;
+	char pressed;
+};
+
+DataArray(KeyboardKey, KeyboardKeys, 0x03B0E3E0, 256);
+
+struct KeyData
+{
+	Uint32 WindowsCode;
+	Uint32 SADX2004Code;
+};
+
+KeyData SADX2004Keys[] = {
```

Comment:
This should be defined in a cpp file and marked as `extern` (e.g. `extern KeyData SADX2004Keys[];`)